SherpaFrontEnd.Tests/Pages/SurveyQuestionsTest.cs

```diff
@@ -378,4 +378,92 @@ public void ShouldShowUnableToSubmitSurveyResponseWhenServiceThrowsError()
         var contactCoachElement = appComponent.FindElementByCssSelectorAndTextContent("p", "Please contact your coach");
         Assert.NotNull(contactCoachElement);
     }
+
+
+    [Fact]
+    public async Task ShouldSendAnswerSurveyDtoToSurveyServiceAndShowThankYouMessageInSpanishOnSubmit()
```

Comment:
Maybe this test can be moved to the component you created in a later commit. Also from the component you shouldn't need so much set up